common/src/main/java/com/revenuecat/purchases/common/APIKeyValidator.kt

```diff
@@ -0,0 +1,64 @@
+package com.revenuecat.purchases.common
+
+import androidx.annotation.VisibleForTesting
+import com.revenuecat.purchases.Store
+import com.revenuecat.purchases.strings.ConfigureStrings
+
+private const val GOOGLE_API_KEY_PREFIX = "goog_"
+private const val AMAZON_API_KEY_PREFIX = "amzn_"
+
+class APIKeyValidator {
```

Comment:
I need to double check something with you... If we don't make this class internal, is it accessible for devs? I don't think so right? They would have to do `implementation` on the common library, or we would have to be doing `api` on `common` from the `purchases` module, right? 

Sorry I am asking this but I always get confused haha

Reply:
No worries! And that's correct, devs shouldn't have access to this class since `purchases` uses `implementation` on `common`. Devs shouldn't have access to it unless we used `api` instead of `implementation` on the purchases module.